specs/language/conversions.tex

```diff
@@ -130,6 +143,42 @@
 prvalue of type \texttt{matrix<T,x,y>}. The destination value is the source
 value replicated into each element of the destination.
 
+\p A glvalue of type \texttt{vector<T,1>} can be converted to a cxvalue of type
+\texttt{vector<T,x>} or a prvalue of type \texttt{vector<T,1>} can be converted
+to a prvalue of type \texttt{vector<T,x>}. The destination value is the value in
+the source vector replicated into each element of the destination.
+
+\Sec{Array and Class splat conversion}{Conv.asplat}
+
+\p A glvalue of type \texttt{T} can be converted to a cxvalue of type
```

Comment:
I believe that this conversion can only produce a prvalue, not a cxvalue. The only place cxvalues are created is for `inout` parameters, and I believe this code is invalid:

```hlsl
struct S {
    float V[4];
};

void fn(inout float4 F) {}

export void call() {
    S V = {1.xxxx};
    fn((float4)V);
    fn(V);
}
```
see: https://godbolt.org/z/Ms4ocrGMf